include/antidote.hrl

```diff
@@ -137,9 +142,33 @@
 
 -define(CLOCKSI_TIMEOUT, 1000).
 
--record(transaction, {snapshot_time :: snapshot_time(),
-                      vec_snapshot_time,
-                      txn_id :: txid()}).
+-record(operation_payload, {
+    key :: key(),
+    type :: type(),
+    op_param :: op(),
+    snapshot_vc :: vectorclock(),
+    dependency_vc :: vectorclock(),
+    dc_and_commit_time :: commit_time(),
+    txid :: txid()}).
+
+-record(transaction, {
+    snapshot_clock :: snapshot_time(),
+    transactional_protocol :: atom(),
```

Comment:
Defined a type transactional_protocol() in antidote.hrl that can be of physics | clocksi | gr